.github/workflows/e2e_tests.yml

```diff
@@ -1,47 +1,183 @@
-name: E2E tests (master/release branch against staging/preprod)
+# several test modes:
+# - MASTER_STAGING: master branch against staging
+# - RELEASE_PREPROD: latest release branch against preprod
+# - MASTER_LTS: master branch against LTS
+# - PREV_RELEASE_TESTS_STAGING: SDK from master, previous release tests against staging
+# - LTS_CYCLE_FIRST_SDK: test with first SDK of LTS cycle, against staging
+
+name: E2E tests
 on:
   workflow_dispatch:
+    inputs:
+      runE2ETests:
+        description: "Run E2E tests?"
+        required: true
+        type: boolean
+        default: true
+      runNotebookTests:
+        description: "Run notebook tests?"
+        required: true
+        type: boolean
+        default: true
+      testMode:
+        description: "Test mode: MASTER_STAGING (master branch against staging), RELEASE_PREPROD (latest release branch against preprod), MASTER_LTS (master branch against LTS), PREV_RELEASE_TESTS_STAGING (SDK from master, previous release tests against staging), LTS_CYCLE_FIRST_SDK (test with first SDK of LTS cycle, against staging)"
+        required: true
+        type: choice
+        default: "MASTER_STAGING"
+        options:
+          - MASTER_STAGING
+          - RELEASE_PREPROD
+          - MASTER_LTS
+          - PREV_RELEASE_TESTS_STAGING
+          - LTS_CYCLE_FIRST_SDK
```

Comment:
can we have the option to launch all of them? I feel that it would be the done to use the most

Reply:
I can try to implement that yep

Reply:
done:  https://github.com/kili-technology/kili-python-sdk/actions/runs/4305553000